### ISSUE TITLE
Dotnet is not installed. Please install dotnet.

### ISSUE THREAD

**Issue by DeadiKation** [Kwoth/NadekoBot#2482]
Hello, Tried asking for help in the discord but no one could help. My Nadeko installation (In a sub-folder) cannot see my dotnet installation. I've already re-installed both dotnet and Nadeko, including a restart of my box. If it helps I did the workaround for Ubuntu 18.04 previously with the export path solution. I've since removed that workaround but yet my installation of Nadeko still does not see dotnet. Dotnet was also recently re-installed twice and shown to be working. I also checked if Nadeko worked from the home folder itself following direct instructions and it did. However I'm attempting to run it from this sub-folder.

Provided below is my terminal session.

![image](https://user-images.githubusercontent.com/4283883/43671073-879628b8-9748-11e8-9822-fda02c7d6a3a.png)

**Comment by shikhir-arora** [comment]
Have you tried:

`cd ~ && wget -N https://github.com/Kwoth/NadekoBot-BashScript/raw/1.9/linuxAIO.sh && bash linuxAIO.sh`

**Comment by Kwoth** [comment]
Install prerequisites. Check some guide online on how to install dotnet core sdk 2.1 on your distro. You didn't prove to us that your dotnet is installed.

Type `dotnet --version` and see if it works.

**Comment by DeadiKation** [comment]
@shikhir-arora I have tried that, hence my explanation of "I also checked if Nadeko worked from the home folder itself following direct instructions and it did. However I'm attempting to run it from this sub-folder."

@Kwoth I believe the above explanation proves that it's installed and working since the bot actually works from the home folder. However I've fixed this issue by symlinking dotnet from `/usr/share/dotnet/dotnet` to `/usr/local/bin/` - Apparently the workaround I performed for Ubuntu 18.04 on the Discord messed with that, and even after re-install did not re-create this.

`sudo ln -s /usr/share/dotnet/dotnet /usr/local/bin/`
